oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+server-tools https://github.com/laslabs/server-tools bugfix/10.0/base_multi_image-hook-revisited
```

Comment:
Yeah please don't merge yet. I'm just using this as proof that my patch in server-tools actually works this time. I didn't do it last time & now we're in this situation. Sorry, I should have been more clear in my comment.